web/packages/teleterm/src/ui/Search/pickers/results.story.tsx

```diff
@@ -343,68 +343,89 @@ const SearchResultItems = () => {
-    onPick={() => {}}
-    onBack={() => {}}
-    render={() => null}
-    attempts={[]}
-    addWindowEventListener={() => ({ cleanup: () => {} })}
-    ExtraTopComponent={
-      <>
-        <NoResultsItem
-          clustersWithExpiredCerts={new Set()}
-          getClusterName={routing.parseClusterName}
-        />
-        <NoResultsItem
-          clustersWithExpiredCerts={new Set([clusterUri])}
-          getClusterName={routing.parseClusterName}
-        />
-        <NoResultsItem
-          clustersWithExpiredCerts={new Set([clusterUri, '/clusters/foobar'])}
-          getClusterName={routing.parseClusterName}
-        />
-        <ResourceSearchErrorsItem
-          getClusterName={routing.parseClusterName}
-          showErrorsInModal={() => window.alert('Error details')}
-          errors={[
-            new ResourceSearchError(
-              '/clusters/foo',
-              'server',
-              new Error(
-                '14 UNAVAILABLE: connection error: desc = "transport: authentication handshake failed: EOF"'
-              )
-            ),
-          ]}
-        />
-        <ResourceSearchErrorsItem
-          getClusterName={routing.parseClusterName}
-          showErrorsInModal={() => window.alert('Error details')}
-          errors={[
-            new ResourceSearchError(
-              '/clusters/bar',
-              'database',
-              new Error(
-                '2 UNKNOWN: Unable to connect to ssh proxy at teleport.local:443. Confirm connectivity and availability.\n	dial tcp: lookup teleport.local: no such host'
-              )
-            ),
-            new ResourceSearchError(
-              '/clusters/foo',
-              'server',
-              new Error(
-                '14 UNAVAILABLE: connection error: desc = "transport: authentication handshake failed: EOF"'
-              )
-            ),
-          ]}
-        />
-        <SuggestionsError
-          statusText={
-            '2 UNKNOWN: Unable to connect to ssh proxy at teleport.local:443. Confirm connectivity and availability.\n	dial tcp: lookup teleport.local: no such host'
-          }
-        />
-        <TypeToSearchItem hasNoRemainingFilterActions={false} />
-        <TypeToSearchItem hasNoRemainingFilterActions={true} />
-      </>
-    }
-  />
-);
+const AuxiliaryItems = () => {
+  const [advancedSearchEnabled, setAdvancedSearchEnabled] = useState(false);
+  const advancedSearch = {
+    isToggled: advancedSearchEnabled,
+    onToggle: () => setAdvancedSearchEnabled(prevState => !prevState),
+  };
+
+  return (
+    <ResultList<string>
+      onPick={() => {}}
+      onBack={() => {}}
+      render={() => null}
+      attempts={[]}
+      addWindowEventListener={() => ({
+        cleanup: () => {},
+      })}
+      ExtraTopComponent={
+        <>
+          <NoResultsItem
+            clustersWithExpiredCerts={new Set()}
+            getClusterName={routing.parseClusterName}
+            advancedSearch={advancedSearch}
```

Comment:
> I decided to add the `AdvancedSearchToggle` to each item (in opposition to injecting JSX into them), this will ensure that the items look the same in storybook, as they do in the app.

That's true but now each of those `Item` components needs to know what props `StyledAdvancedSearchToggle` accepts. We end up having to pass some extra stuff in the story anyway so I feel like passing the search toggle component as a prop or a child would've been marginally better. Not a blocker though.